tutorials/international/index.md

```diff
@@ -33,7 +33,7 @@ can also be internationalized using the same classes and logic.
 <aside class="alert alert-info" markdown="1">
 **Sample Internationlized Apps**<br>
 
-If you'd like to start out by reading the code for a internationalized Flutter app, here are two small samples. The first one is intended to be as simple as possible, and the second one uses the APIs and tools provided by the [intl](https://pub.dartlang.org/packages/intl) package.
+If you'd like to start out by reading the code for an internationalized Flutter app, here are two small examples. The first one is intended to be as simple as possible, and the second one uses the APIs and tools provided by the [intl](https://pub.dartlang.org/packages/intl) package.
```

Comment:
we prefer breaking lines at 80 cols.

It also makes reviews easier, by making it easier to leave a comment closer to the relevant text.